test/unittest/client/session/stream/InputStreamTest.cpp

```diff
@@ -41,22 +41,25 @@ class NoneInputStreamTest : public ::testing::Test
 TEST_F(NoneInputStreamTest, PushMessage)
 {
     uint8_t buf[128] = {0};
-    InputMessagePtr input_message(new InputMessage(buf, sizeof(buf)));
+    InputMessagePtr input_message;
 
     for (uint16_t i = 0; i < BEST_EFFORT_STREAM_DEPTH; ++i)
     {
+        input_message.reset(new InputMessage(buf, sizeof(buf)));
```

Comment:
Maybe NoneInputStream should provide the same API as std container it is wrapping:

This two lines would become (or something nicer):

`none_stream_.emplace(new InputMessage {buf, sizeof(buf)});`
or
`none_stream_.push_message(std::make_unique<InputMessage>(buf, sizeof(buf)), 0xFFFF);`
in case we do care about the second parameter (It is really used?).

Reply:
I hope you like it